lib/krumo/class.krumo.php

```diff
@@ -1099,7 +1099,7 @@
 ?>
 <li class="krumo-child">
 
-  <div class="krumo-element<?php echo count($data) > 0 ? ' krumo-expand' : '';?>"
+  <div class="krumo-element<?php echo ' krumo-expand';?>">
```

Comment:
The PHP tags here seem irrelevant now...

Reply:
Makes perfect sense 👍 ...done!